src/error.ts

```diff
@@ -16,6 +16,9 @@
 // HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 // OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+// rdfjs/types package throws linter errors
+// eslint-disable-next-line import/no-unresolved
```

Comment:
That's weird, never seen that happen elsewhere, might check it out.

Reply:
Yeah, it's super weird. It doesn't happen in solid-client (the only library I'm aware of which is using this) because it doesn't have the dependency linter.

Reply:
I'll try to look at it before we make public.

Reply:
This can be fixed using the docs here: https://github.com/import-js/eslint-plugin-import/issues/1485

![Screenshot 2021-12-15 at 19 29 29](https://user-images.githubusercontent.com/30827/146244417-2ad50f83-7a68-4844-9034-d3defb579c39.png)

Reply:
So we need to fix this in our base eslint config.

Reply:
I've said this before, I think that we should consider making our eslint-config-base peerDependencies non version specific (all *):
https://github.com/inrupt/javascript-style-configs/blob/c169edebb71650bd059e3bdc3fe990cb3cb008c5/eslint-config-base/package.json#L23-L34

Reply:
@matthieubosquet because we're using `^` versions in our peerDependencies, nothing's stopping us from updating. The eslint config only specifies a minimum, which _is_ useful.

Reply:
Ok, I get that, but it makes us stuck on majors.

What about just starring the two @typescript-eslint dependencies?

Reply:
We *do* want to be on the same majors across our libraries, or as close as possible.